harwest/harwest.py

```diff
@@ -27,6 +28,19 @@ def build_argument_parser():
     'leetcode', help="Scrape solutions from the leetcode platform")
   lc_parser.set_defaults(func=leetcode)
 
+
```

Comment:
I'm using the PEP8 styling convention for Python with the exception of using 2 spaces over 4 spaces. I've configured the linter to use the same on my IDE (Intellij) which helps a lot in quickly identifying and fixing formatting issues. Would recommend setting up the same in the IDE of your choice and fixing some of the formatting issues which might have cropped up in these changes.

Reply:
sure, i'll use sublime. i'll configure it accordingly